lib/tests/testdata/ebpf_sample.c

```diff
@@ -0,0 +1,12 @@
+#include "uapi/linux/bpf.h"
+
+#ifndef __section
```

Comment:
Now that we have ebpf/utils.h, we should re-use it everywhere we can, rather than redefine __section.

Reply:
I got rid of this and started using lib/ebpf/sample.c itself